versions.tf

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2018 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
The version should be pinned to `~> 0.12.6` to include support of `for_each` and to restrict support of 0.13.